docs/source/terms_and_model.rst

```diff
@@ -1575,6 +1497,84 @@ ISCN guidelines [1]_.
 Deprecated and Obsolete Classes
 @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
 
+.. _SimpleInterval:
+
+SimpleInterval
+##############
+
+**Computational Definition**
+
+DEPRECATED: A SimpleInterval represents a span of sequence. Positions are
```

Comment:
Question: Will we modify our normalization routine to convert `SimpleInterval`s to `SequenceInterval`s? I suppose that would be a breaking change with what's already released. So sort of a dumb question I guess.
This is an interesting issue which we may want to call out more clearly as it does represent two ways to represent the same intervals.

Reply:
That's an interesting idea, but I'd recommend against that implicit behavior.

Instead, I think we should plan on an explicit conversion process for SimpleInterval → SequenceInterval and SequenceState → LiteralSequenceExpression.